packages/flutterfire_cli/lib/src/commands/upload_symbols.dart

```diff
@@ -359,7 +359,8 @@ class UploadCrashlyticsSymbols extends FlutterFireCommand {
       ],
       environment: {
         'PLATFORM_NAME': envPlatformName,
-        'CONFIGURATION': envConfiguration,
+        // Hard code "Release" to ensure "App.framework.dsym" is uploaded and complies with upload-symbol script
+        'CONFIGURATION': 'Release',
```

Comment:
Let's doing a check, if `envConfiguration` contain `debug` word we pass `Debug`, otherwise Release. I don't think debug mode produces framework dsym.

Reply:
Also want to double check how customizable for user to change Configuration name for flutter? I see for flutter application you can change the config name quiet easily, but under properties for native app configuration there are only debug and release options. If we allow user to change configuration name by themselves it could easily break the current check for `Release` and `Debug` mode.
<img width="665" alt="Screenshot 2024-02-29 at 2 50 42 PM" src="https://github.com/invertase/flutterfire_cli/assets/16548721/03e78cc9-4bf9-47ae-8a2b-528b373e6828">
?

Reply:
As mentioned in the chat, this solution is more robust than checking the name of the build configuration.